### PR TITLE
Completed zh-1.3.0

### PR DESCRIPTION
**Pull request by pinggod**

### CODE REVIEW

File: zh/_architecture.md [hunk 1]
```diff
@@ -185,17 +201,20 @@
 
 {% include snippets/architecture/03/index.html %}
 
-<div class="note">
-  <p>为了不用亲自引入每一个文件，有一个叫做 <a href="https://github.com/chriseppstein/sass-globbing">sass-globbing</a> 的 Ruby Sass 扩展程序，使在 Sass 的 <code>@import</code> 中,使其做为 glob 模式，就像这样：<code>@import "components/*"</code></p>
-  <p>话虽如此，却不推荐它，因为它按照字母顺序引入文件，这往往并不是想要的，特别是处理一个对源文件顺序有所依赖的编程语言的时候。</p>
-</div>
+## 关于 Golobbing
```
Comment (HugoGiraudel) [inline zh/_architecture.md]: Should it be *Globbing* or is *Golobbing* correct?

File: zh/_author.md [hunk 2]
```diff
@@ -1,10 +1,15 @@
...
 
-[Hugo Giraudel](http://hugogiraudel.com)，23 岁，来自法国的前端工程师，现居柏林，目前在 [Edenspiekermann](http://edenspiekermann.com) 工作。
+[Hugo Giraudel](http://hugogiraudel.com)，24 岁，来自法国的前端工程师，现居柏林，目前在 [Edenspiekermann](http://edenspiekermann.com) 工作。
 
 拥有多年的 Sass 开发经验，开发的项目包括：[SassDoc](http://sassdoc.com), [SitePoint Sass Reference](http://sitepoint.com/sass-reference/) 和 [Sass-Compatibility](http://sass-compatibility.github.io)。更多信息，请查阅这里的[项目列表](http://github.com/HugoGiraudel/awesome-sass)。
 
 已出版图书 [《CSS3 Pratique du Design Web》](http://css3-pratique.fr/)（法语版）和[《Jump Start Sass》](https://learnable.com/books/jump-start-sass)（英语版）。
 
-{% include partials/twitter.html %}
+<div class="button-wrapper">
```
Comment (HugoGiraudel) [inline zh/_author.md]: It should be `{% include partials/twitter.html %}` here. I can still fix it myself though.

File: zh/_contributing.md [hunk 3]
```diff
@@ -1,10 +1,20 @@
 
 # 贡献
 
-Sass指南是我利用空暇时间维持的一个免费项目。不用多说,相信你也能够理解，持续更新、编写文档和其他相关事宜是一件工作量不小的事情。幸运的是，我从许多非常棒的贡献者那里得到了帮助，从而让该规范翻译成了<button type="button" data-modal-show="options-panel" class="link-like">多种语言</button>。请向他们致敬！
+Sass指南是我利用空暇时间维持的一个免费项目。不用多说,相信你也能够理解，持续更新、编写文档和其他相关事宜是一件工作量不小的事情。幸运的是，我从许多非常棒的贡献者那里得到了帮助，从而让该规范翻译成了<label for="aside-toggle" class="link-like">多种语言</label>。请向他们致敬！
```
Comment (HugoGiraudel) [inline zh/_contributing.md]: It should be a button here, but I can also fix it myself.

File: zh/_contributing.md [hunk 4]
```diff
@@ -1,10 +1,20 @@
...
 
 现在，如果你想要为它做些贡献，一个很好的建议是：使用社交网络分享它，也可以在 [GitHub repository](https://github.com/HugoGiraudel/sass-guidelines) 上提交修正错误的 issue 或者 pull-request。
 
 在开始之前最后但不是最终的一句话：如果你喜欢这份样式指南，如果它对你或你的团队大有裨益，请考虑支持它，从而鼓励我持续迭代不断改进本文！
 
-{% include partials/contributing.html %}
+<div class="button-wrapper">
```
Comment (HugoGiraudel) [inline zh/_contributing.md]: It should be `{% include partials/contributing.html %}` here but I can also fix it myself.

File: zh/_errors.md [hunk 5]
```diff
@@ -32,3 +26,5 @@
 举个例子，假设你创建了一个 getter 函数来从特定 map 中获取值。如果想要获取的值并不在 map 中，就可能会抛出错误。
 
 {% include snippets/errors/02/index.html %}
+
+更多有关如何高效使用 `@error` 的信息可以点击这一链接：[this introduction about error handling](http://webdesign.tutsplus.com/tutorials/an-introduction-to-error-handling-in-sass--cms-19996)。
```
Comment (HugoGiraudel) [inline zh/_errors.md]: Why is the label of the link non-translated?

File: zh/_extend.md [hunk 6]
```diff
@@ -15,7 +15,7 @@
 
 当继承 `.class` 或 `%placeholder` 时，如果父类和子类是同一类型，那么建议只使用 `@extend` 来实现，比如 `.error` 是 `.warning` 的一种，那么 `.error` 就可以通过 `@extend .warning` 来实现。
 
-{% include snippets/extend/01/index.html %}
++{% include snippets/extend/01/index.html %}
```
Comment (HugoGiraudel) [inline zh/_extend.md]: One too many + here. :)

File: zh/_extend.md [hunk 7]
```diff
@@ -25,25 +25,43 @@
 - 避免继承常见的父类选择器（比如： `.foo .bar`）或者是常见的相邻选择器（比如：`.foo ~ .bar`），否则会让选择器的数量急速增加。
 
 <div class="note">
-  <p>通常来说，<code>@extend</code> 有助于减少文件体积大小，因为它的操作本质上是合并选择器而不是赋值样式。话虽如此，当你使用 <a href="http://en.wikipedia.org/wiki/Gzip">Gzip</a> 压缩文件时，<code>@extend</code> 对于文件压缩的好处几乎是可以忽略的。</p>
-  <p>这也就是说，如果你不能使用类似 Gzip 的工具，那么就可以考虑使用 <code>@extend</code> 来减少不必要的重复，特别是当样式文件的大小成为性能瓶颈的时候，这种方式尤为有效。</p>
+    <p>通常来说，<code>@extend</code> 有助于减少文件体积大小，因为它的操作本质上是合并选择器而不是赋值样式。话虽如此，当你使用 <a href="http://en.wikipedia.org/wiki/Gzip">Gzip</a> 压缩文件时，<code>@extend</code> 对于文件压缩的好处几乎是可以忽略的。</p>
```
Comment (HugoGiraudel) [inline zh/_extend.md]: Too large indentation.

File: zh/_extend.md [hunk 8]
```diff
@@ -25,25 +25,43 @@
...
+    <p>这也就是说，如果你不能使用类似 Gzip 的工具，那么就可以考虑使用 <code>@extend</code> 来减少不必要的重复，特别是当样式文件的大小成为性能瓶颈的时候，这种方式尤为有效。</p>
 </div>
 
-## 继承和媒体查询
+### 继承和媒体查询
```
Comment (HugoGiraudel) [inline zh/_extend.md]: One too many hashes here.

File: zh/_extend.md [hunk 9]
```diff
@@ -25,25 +25,43 @@
...
+  <p>感谢 <a href="https://twitter.com/davidkpiano">David Khourshid</a> 对本节提供的技术支持。</p>
 </div>
 
-###### 扩展阅读
 
-* [What Nobody Told you About Sass Extend](http://www.sitepoint.com/sass-extend-nobody-told-you/)
-* [Why You Should Avoid Extend](http://www.sitepoint.com/avoid-sass-extend/)
-* [Don’t Over Extend Yourself](http://pressupinc.com/blog/2014/11/dont-overextend-yourself-in-sass/)
-* [When to Use Extend; When to Use a Mixin](http://csswizardry.com/2014/11/when-to-use-extend-when-to-use-a-mixin/)
+
```
Comment (HugoGiraudel) [inline zh/_extend.md]: All the useless empty lines here. :P

File: zh/_introduction.md [hunk 10]
```diff
@@ -48,8 +48,4 @@
 
 点击这里可以查看位于 [SassDoc repository](https://github.com/SassDoc/sassdoc/blob/master/GUIDELINES.md) 上的一个 Styleguide 扩展：
 
-> 这 是一个由 Felix Geisendörfer 开发的 [Node Styleguide](https://github.com/felixge/node-style-guide) 扩展。这份文档全面覆盖了 Node Styleguide 的内容。
-
-###### 扩展阅读
-
-* [Keep Sass Simple](http://www.sitepoint.com/keep-sass-simple/)
+> [这](https://github.com/SassDoc/sassdoc/blob/master/GUIDELINES.md) 是一个由 Felix Geisendörfer 开发的 [Node Styleguide](https://github.com/felixge/node-style-guide) 扩展。这份文档全面覆盖了 Node Styleguide 的内容。
```
Comment (HugoGiraudel) [inline zh/_introduction.md]: The first link should be removed.

File: zh/_loops.md [hunk 11]
```diff
@@ -1,7 +1,7 @@
 
 # 循环
 
-因为Sass提供了如 [lists](#section-29) 和 [maps](#maps) 这样的复杂数据结构，所以对于提供给开发者遍历这些数据结构的能力也无需惊讶。
+因为Sass提供了如 [lists](#lists) 和 [maps](#maps) 这样的复杂数据结构，所以对于提供给开发者遍历这些数据结构的能力也无需惊讶。
```
Comment (HugoGiraudel) [inline zh/_loops.md]: The hash was actually correct as far as I can tell.

File: zh/_mixins.md [hunk 12]
```diff
@@ -17,10 +17,7 @@
 
 {% include snippets/mixins/02/index.html %}
 
-###### 扩展阅读
-
-* [Sass Mixins to Kickstart your Project](http://www.sitepoint.com/sass-mixins-kickstart-project/)
-* [A Sass Mixin for CSS Triangles](http://www.sitepoint.com/sass-mixin-css-triangles/)
+更多复杂示例可以参考：[this mixin to generate CSS triangles](http://www.sitepoint.com/sass-mixin-css-triangles/), [this mixin to create long shadows](http://www.sitepoint.com/ultimate-long-shadow-sass-mixin/) 和 [this mixin to polyfill CSS gradients for old browsers](http://www.sitepoint.com/building-linear-gradient-mixin-sass/)。
```
Comment (HugoGiraudel) [inline zh/_mixins.md]: Why are links label non translated?

File: zh/_mixins.md [hunk 13]
```diff
@@ -40,10 +37,8 @@ Sass 的混合宏和函数声明非常智能，你只需给函数/混合宏一
 
 {% include snippets/mixins/04/index.html %}
 
-###### 扩展阅读
-
-* [Sass Multiple Arguments, Lists or Arglist](http://www.sitepoint.com/sass-multiple-arguments-lists-or-arglist/)
-
+更多有关多参数、列表参数的信息请参考这篇文章：[SitePoint has a nice piece on the topic](http://www.sitepoint.com/sass-multiple-arguments-lists-or-arglist/)。
```
Comment (HugoGiraudel) [inline zh/_mixins.md]: Why is this link label non translated?

File: zh/_mixins.md [hunk 14]
```diff
@@ -17,10 +17,7 @@
 
 {% include snippets/mixins/02/index.html %}
 
-###### 扩展阅读
-
-* [Sass Mixins to Kickstart your Project](http://www.sitepoint.com/sass-mixins-kickstart-project/)
-* [A Sass Mixin for CSS Triangles](http://www.sitepoint.com/sass-mixin-css-triangles/)
+更多复杂示例可以参考：[this mixin to generate CSS triangles](http://www.sitepoint.com/sass-mixin-css-triangles/), [this mixin to create long shadows](http://www.sitepoint.com/ultimate-long-shadow-sass-mixin/) 和 [this mixin to polyfill CSS gradients for old browsers](http://www.sitepoint.com/building-linear-gradient-mixin-sass/)。
 
 ## Argument-less mixins
```
Comment (HugoGiraudel) [inline zh/_mixins.md]: Title non translated?

File: zh/_naming.md [hunk 15]
```diff
@@ -27,22 +23,18 @@
 
 {% include snippets/naming/02/index.html %}
 
-###### 扩展阅读
-
-* [Dealing With Constants in Sass](http://www.sitepoint.com/dealing-constants-sass/)
+如果你在 Sass 中使用常量，请参考这篇文章：[dealing constants sass](http://www.sitepoint.com/dealing-constants-sass/).
```
Comment (HugoGiraudel) [inline zh/_naming.md]: Link label non translated?

File: zh/_syntax.md [hunk 16]
```diff
@@ -347,7 +311,9 @@ Sass 中一个正在被众多开发者滥用的功能，就是**选择器嵌套*
 
 选择器越具体则声明语句越冗长，而且对最近选择器的引用(`&`)也越频繁。在某些时候，出现混淆选择器路径和探索下一级选择器的错误率很高，这非常不值得。
 
-为了防止此类情况，我们**应该尽可能避免选择器嵌套**。然而，显然只有少数情况适应这一措施。
+为了防止此类情况，我们今年就 [the Inception rule](http://thesassway.com/beginner/the-inception-rule) 讨论了很多很多。它建议嵌套不要超过三层，我的一件比较激进，**建议尽量避免使用嵌套**。 
+
+虽然我们在下一节看到这条规则有一些例外，但这一观点还是很受欢迎的。更多信息请阅读：[Beware of Selector Nesting](http://www.sitepoint.com/beware-selector-nesting-sass/) 和 [Avoid nested selectors for more modular CSS](http://thesassway.com/intermediate/avoid-nested-selectors-for-more-modular-css).
```
Comment (HugoGiraudel) [inline zh/_syntax.md]: Links label non translated?

File: zh/_tldr.md [hunk 17]
```diff
@@ -5,39 +5,39 @@
 
 ## 核心原则
 
-* 创建编程规范的目的就是为了保证协作开发的一致性。即使你对本文有不同的意见，也要保证整体开发的一致性。[↩](#section-8)
-* 尽可能让 Sass 代码保持简洁。除非是绝对需要，否则绝没有必要构建复杂的系统。[↩](#section-10)
-* 请记住，有时候保持简洁（KISS，Keep It Simple, Stupid）比减少重复（Don't Repeat Yourself）更重要。[↩](#section-10)
+* 创建编程规范的目的就是为了保证协作开发的一致性。即使你对本文有不同的意见，也要保证整体开发的一致性。[↩](#why-a-styleguide)
```
Comment (HugoGiraudel) [inline zh/_tldr.md]: I corrected all the links in this section, please discarded your changes.

File: zh/_tools.md [hunk 18]
```diff
@@ -60,14 +43,9 @@
 
 {% include snippets/tools/01/index.html %}
 
+如果你不确信 SCSS-lint 的必要性，建议阅读以下文章：[Clean Up your Sass with SCSS-lint](http://blog.martinhujer.cz/clean-up-your-sass-with-scss-lint/), [Improving Sass code quality on theguardian.com](http://www.theguardian.com/info/developer-blog/2014/may/13/improving-sass-code-quality-on-theguardiancom) 和 [An Auto-Enforceable SCSS Styleguide](http://davidtheclark.com/scss-lint-styleguide/).
```
Comment (HugoGiraudel) [inline zh/_tools.md]: Links label non translated?

File: zh/index.md [hunk 19]
```diff
@@ -1,7 +1,6 @@
 ---
 layout: default
 language: zh
```
Comment (HugoGiraudel) [inline zh/index.md]: Please add `direction` back.

File: zh/index.md [hunk 20]
```diff
@@ -1,7 +1,6 @@
...
-direction: ltr
 ---
 
-{% include layout/chapters.html %}
+{% include chapters.html %}
```
Comment (HugoGiraudel) [inline zh/index.md]: Please discard this change.

**Comment by HugoGiraudel** [comment]
Hey there! Thank you so much for doing this. Super happy to see you’ve already updated your translation. I added a few comments if you don’t mind? :)

File: zh/_syntax.md (hunk 16 above)
Comment (pinggod) [inline zh/_syntax.md]: No appropriate chinese words to describe "the Inception rule" here.

Other labels will be translated.

File: zh/_tldr.md (hunk 17 above)
Comment (pinggod) [inline zh/_tldr.md]: I can't find the corrected links.

Comment (HugoGiraudel) [inline zh/_tldr.md]: I mean that `section-8` and such should be correct. I’ll double check everything anyway, so feel free to discard your changes on this file. I’ll make it work. ;)

**Pull request closed without merging**